ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -75,6 +76,7 @@ conformance_test(
         5031,
         5041,
     ],
+    preview_mod_flag = "-C _shared.participant_parameters.unsafe-enable-daml-lf-dev-version=true",
```

Comment:
I'm not sure this is enough see https://www.canton.io/docs/dev/user-manual/usermanual/FAQ.html#how-do-i-enable-unsupported-features